cfgov/jinja2/v1/_includes/blocks/feedback.html

```diff
@@ -11,11 +11,16 @@
 
 <form method="post"
       action="."
+{% if not value.was_it_helpful_text %}
+      class="o-form">
+{% else %}
       class="o-form
              block
              block__bg
              block__border
              block__padded-top">
+{% endif %}
+<div class="form-l">
     <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
     <input type="hidden" name="form_id" value="{{ form_id }}">
     <input type="hidden" name="referrer" value="{{ request.META.HTTP_REFERER }}">
```

Comment:
If you navigate to this page directly, and there's no referrer, then the form element looks like this:

![image](https://cloud.githubusercontent.com/assets/654645/20538796/b2746376-b0c0-11e6-9bdf-d03e78070b61.png)

To avoid this, can [this line](https://github.com/cfpb/cfgov-refresh/pull/2544/files#diff-0acfcc6db8ff7440bb3c2b303a8e3a4bL21) be changed to something like:

```html
<input type="hidden" name="referrer" value="{{ request.META.get('HTTP_REFERER') or '' }}">
```